common/persistence/serialization/thrift_mapper_test.go

```diff
@@ -131,6 +131,22 @@ func TestDomainInfo(t *testing.T) {
 	assert.Equal(t, expected.LastUpdatedTimestamp.Sub(actual.LastUpdatedTimestamp), time.Duration(0))
```

Comment:
Could you update `TestDomainInfo` to compare the new field?

Reply:
👍